charts/kubernetes-agent/templates/_helpers.tpl

```diff
@@ -72,6 +72,15 @@ Create the name of the pod cluster role for deleting pods
 {{- printf "%s-delete-role" (include "kubernetes-agent.scriptPodServiceAccountFullName" .) }}
 {{- end }}
 
+
+{{/*
+Create the name of the pod (namespaced) role for creating pods, and updating tentacle
+*/}}
+{{- define "kubernetes-agent.scriptPodWorkerRoleName" -}}
+{{- printf "%s-worker-role" (include "kubernetes-agent.scriptPodServiceAccountFullName" .) }}
```

Comment:
Minor: this doesn't protect against names >63 characters - the trunc is in the `kubernetes-agent.scriptPodServiceAccountFullName` and so we may have names that are invalid here. This is a problem already so happy to ignore for this PR.